sa/sa.go

```diff
@@ -552,9 +553,11 @@ func (ssa *SQLStorageAuthority) NewOrderAndAuthzs(ctx context.Context, req *sapb
 			// ID and Created were auto-populated on the order model when it was inserted.
 			Id:        order.ID,
 			CreatedNS: order.Created.UnixNano(),
+			Created:   timestamppb.New(order.Created),
 			// These are carried over from the original request unchanged.
 			RegistrationID: req.NewOrder.RegistrationID,
 			ExpiresNS:      req.NewOrder.ExpiresNS,
+			Expires:        req.NewOrder.Expires,
```

Comment:
This is reading a field.

Reply:
I assume the same is true for the `Created` field on line 556?

Reply:
No, that line is reading from an `orderModel` object (populated on line 500), so we know that `order.Created` field exists. The issue here is that `req.NewOrder.Expires` is a brand-new, not-necessarily-populated-yet field.